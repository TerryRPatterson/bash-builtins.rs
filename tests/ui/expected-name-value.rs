use bash_builtins::BuiltinOptions;

#[derive(BuiltinOptions)]
enum Opt {
    #[opt('a')]
    A,
}

fn main() {}
