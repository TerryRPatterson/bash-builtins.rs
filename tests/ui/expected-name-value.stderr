error: invalid #[opt] attribute
 --> $DIR/expected-name-value.rs:5:5
  |
5 |     #[opt('a')]
  |     ^
